app/main/windows.js

```diff
@@ -20,10 +20,10 @@ function buildSplashWindow () {
 function buildSessionWindow () {
   const window = new BrowserWindow({
     show: false,
-    width: 1280,
-    height: 800,
-    minWidth: 1000,
-    minHeight: 700,
+    width: 1100,
```

Comment:
Can we somewhere document these limitations?

Reply:
Added a new subsection in the readme